data/json/monstergroups/misc.json

```diff
@@ -170,7 +170,8 @@
       { "monster": "mon_albino_penguin", "freq": 10, "cost_multiplier": 5, "pack_size": [ 3, 9 ] },
       { "monster": "mon_gator", "freq": 40, "cost_multiplier": 2, "pack_size": [ 1, 2 ] },
       { "monster": "mon_nakedmolerat_giant", "freq": 25, "cost_multiplier": 3 },
-      { "monster": "mon_frog_mother", "freq": 10, "cost_multiplier": 3 }
+      { "monster": "mon_frog_mother", "freq": 10, "cost_multiplier": 3 },
+      {"monster": "mon_mole_cricket", "freq": 30, "cost_multiplier": 3}
```

Comment:
Mole crickets don't like water as far as I know, so they make little sense here as well. Naked mole rats will also move out of the sewers soon, for what it's worth.

Reply:
I see. Mostly crickets were put in the same places as naked mole rats. So I'll remove these too.